plugins/src/main/java/com/google/fhir/gateway/plugin/ListAccessChecker.java

```diff
@@ -327,6 +331,7 @@ private BundlePatients createBundlePatients(RequestDetailsReader requestDetails)
 
     Set<String> patientsToCreate = Sets.newHashSet();
     Set<String> patientsToUpdate = Sets.newHashSet();
+    Set<String> patientsToDelete = patientsInBundleUnfiltered.getDeletedPatients();
```

Comment:
Is this another bug you are fixing (I mean other than #45)? If yes, then please consider adding a different issue for it and reference it in this PR's description (no need to create another PR, we can close both issues with one PR).